src/index.html

```diff
@@ -1,14 +1,81 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header>
+      <img 
+      src="images/logo.png"
+      alt="Moyo logo picture"
+      class="moyo-logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item" >
+            <a
+            href=" " 
+            class="nav__link is-active"
+            >APPLE</a>
```

Comment:
```suggestion
            <a
              href="#" 
              class="nav__link is-active"
            >
              Apple
            </a>
```
Не оставляй href пустым, указывай `#` как дефолтный href
Текст тут пиши обычным кеглем, а через css делай uppercase